benchmarks/lockhammer/scripts/lh_cfg.yaml

```diff
@@ -0,0 +1,88 @@
+# CPU Frequency = 2GHz
```

Comment:
What are these numbers supposed to mean?

Reply:
They are used as common assumptions for sweep test default input.